localisation/00_zuwc_united_localisation.csv

```diff
@@ -2886,6 +2886,17 @@ GetHorses_horses;horses;;;;;;;;;;;;;x
 GetEluneTheLightCap_The_Light;The Light;;;;;;;;;;;;;x
 GetPronoun_Her;Her;;;;;;;;;;;;;x
 GetPronoun_Its;Its;;;;;;;;;;;;;x
+average_lifespan;Average Lifespan;;;;;;;;;;;;;x
+average_lifespan_percent;Average Lifespan;;;;;;;;;;;;;x
+text_effect_average_lifespan_immortality;Average Lifespan is close to �GImmortality�!;;;;;;;;;;;;;x
+AUTOCEPHALOUS_PATRIARCHS;Autonomous Bishops;;;;;;;;;;;;;x
+AUTOCEPHALOUS_REL_HEAD_TITLE;Bishop;;;;;;;;;;;;;x
+AUTOCEPHALY_IRO;�YAdditional Autonomous Bishops�!;;;;;;;;;;;;;x
+REL_AUTOCEPHALY_DESC;Autonomous Bishops under Kings;;;;;;;;;;;;;x
+PENTARCHY_DRO;All $RELIGION$ kingdoms or empires also have their own autonomous Bishops. The Court Chaplain of such a kingdom or empire is also considered the Bishop of that realm. He will always do what his liege asks of him, but only the $REL_HEAD_TITLE$ can grant the Invasion Casus Belli. Each autonomous Bishop controls Excommunication within their respective realm.;;;;;;;;;;;;;x
```

Comment:
add german: "Alle $RELIGION$ Königreiche oder Imperien haben ihre eigenen autonomen Bischöfe. Der Hofkaplan eines solchen Königreichs oder Imperiums gilt auch als der Bischof dieses Reiches. Er wird immer das tun, was sein Lehnsherr von ihm verlangt, aber nur der $REL_HEAD_TITLE$ kann den Invasions-Kriegsgrund gewähren. Jeder autonome Bischof kontrolliert die Exkommunikation in seinem jeweiligen Reich."